src/components/Nav/components/NavAvatar/index.tsx

```diff
@@ -2,7 +2,7 @@ import { Avatar, Button, Typography } from '@mui/material';
 import GordonLoader from 'components/Loader';
 import { useUser } from 'hooks';
 import { forwardRef, useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, NavLink } from 'react-router-dom';
```

Comment:
The `Link` import is now unused, isn't it?
```suggestion
import { NavLink } from 'react-router-dom';
```

Reply:
Tis indeed :slightly_smiling_face: , and forwardRef.